src/abstractdataframe/abstractdataframe.jl

```diff
@@ -111,6 +111,8 @@ names!(df::AbstractDataFrame, vals)
 * `df` : the AbstractDataFrame
 * `vals` : column names, normally a Vector{Symbol} the same length as
   the number of columns in `df`
+* `strict` : if names! should disallow duplicate colum names; `true`
```

Comment:
I'd rather call this `allow_duplicates` or something similar, to be clearer.

Also, don't repeat the name of the function in the docs for an argument, that adds noise without any clear benefit. Should also explain what happens when duplicates are present.